src/index.ts

```diff
@@ -246,6 +247,17 @@ program
         }
     });
 
+const devicesGroup = program.command('devices').alias('d').description('Operations on devices');
+
+devicesGroup
+    .command('list-all')
+    .description('Lists all registered devices that can access to your account')
+    .action(listAllDevices);
+devicesGroup
+    .command('remove-all')
```

Comment:
i would recommend having the following commands instead

- `remove -i <identifier>`
- `remove --all` or `remove -a`